integration

```diff
@@ -1021,6 +1021,7 @@
   "neggert/hass-egauge",
   "nelbs/solaredge-forecast",
   "NemesisRE/sensor.plex_recently_added",
+  "NemoN/ha-intex-swg",
```

Comment:
https://github.com/NemoN/ha-intex-swg/blob/1.2.0/README.md
There is no released documentation of what the integration does.